Scripts/script-TaniumApprovePendingActions.yml

```diff
@@ -157,5 +158,4 @@ dependson:
   - tn-add-object
   - tn-get-object
 timeout: 0s
-releaseNotes: "Added description"
-
+releaseNotes: Deprecated script
```

Comment:
please write the command that replace this script.
Deprecated script, use `command name` instead.